packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -465,6 +465,41 @@ describe('FLE tests', () => {
       const compactResult = await shell.executeLine(`autoMongo.getDB('${dbname}').test.compactStructuredEncryptionData()`);
       expect(compactResult).to.include('ok: 1');
     });
+
+    it('creates an encrypted collection and generates data encryption keys automatically per encrypted fields', async() => {
+      const shell = TestShell.start({ args: ['--nodb', `--cryptSharedLibPath=${cryptLibrary}`] });
```

Comment:
This shouldn’t require automatic encryption. We usually omit `--cryptSharedLibPath=...` in those cases, to make sure that this still works even if there is no crypt shared library available (which should only ever be required for automatic encryption):

```suggestion
      const shell = TestShell.start({ args: ['--nodb'] });
```

Reply:
Mh no, this should really work without the shared library present… Maybe you need to set `explicitEncryptionOnly: true` (or `bypassQueryAnalysis: true`, but that would be less desirable because it leaves more features enabled) when creating `keyMongo`, similar to other tests here which don’t use the shared library?

Reply:
Ahh you're right. It works fine with `bypassQueryAnalysis`. I guess I missed a few readings - gonna go back to docs now. Thanks :)